### PR TITLE
Fixing build errors related to popover and legends

### PR DESCRIPTION
**Pull request by srmukher**
Fixing build errors related to popover and legends

### CODE REVIEW

File: packages/react-components/react-charts-preview/src/components/CommonComponents/Popover.types.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import { IChartDataPoint, IYValueHover } from '../../index';
 
 export interface IPopoverComponentProps {
+  id: string;
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/CommonComponents/Popover.types.ts:4]: why is id an interface property

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx [hunk 2]
```diff
@@ -945,10 +943,8 @@ export const VerticalBarChart: React.FunctionComponent<IVerticalBarChartProps> =
       YValueHover: YValueHover,
       hoverXValue: hoverXValue,
     }),
-    gapSpace: 15,
     color: color,
     legend: calloutLegend,
-    XValue: xCalloutValue,
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:951]: why remove this code

File: packages/react-components/react-charts-preview/src/utilities/SVGTooltipText.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { ITooltipHost, ITooltipProps, Tooltip, TooltipDelay } from '@fluentui/react/lib/Tooltip';
-import { Async, KeyCodes, getId, portalContainsElement } from '../Utilities';
+import { Async, KeyCodes, getId, portalContainsElement } from '@fluentui/react/lib/Utilities';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/utilities/SVGTooltipText.tsx:3]: This needs to be removed. Could you comment out the code using all v8 dependencies

File: packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx [hunk 4]
```diff
@@ -28,14 +28,13 @@ export const HorizontalBarChart: React.FunctionComponent<IHorizontalBarChartProp
   let _barHeight: number;
   //let _classNames: IProcessedStyleSet<IHorizontalBarChartStyles>;
   let _uniqLineText: string = '_HorizontalLine_' + Math.random().toString(36).substring(7);
-  let _calloutId: string;
+  let _calloutId: string = useId('_callout');
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx:31]: move this id setting within popover. I see same parameter being used everywhere

File: packages/react-components/react-charts-preview/src/utilities/SVGTooltipText.tsx (hunk 3 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/utilities/SVGTooltipText.tsx:3]: commented the svgTooltip component as discussed

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx (hunk 4 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/components/HorizontalBarChart/HorizontalBarChart.tsx]: moved

File: packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx (hunk 2 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx:951]: this is not used anywhere within popover

**Pull request merged**